src/device/pf_port.c

```diff
@@ -101,13 +101,13 @@ const pnet_port_cfg_t * pf_port_get_config (pnet_t * net, int loc_port_num)
 uint16_t pf_port_loc_port_num_to_dap_subslot (int loc_port_num)
```

Comment:
These functions should have tests (or regression tests for the problem being fixed).

Reply:
Tests added.
Done